rust/src/sip/detect.rs

```diff
@@ -493,7 +496,7 @@ pub unsafe extern "C" fn SCDetectSipRegister() {
         url: String::from("/rules/sip-keywords.html#sip-from"),
         setup: sip_from_hdr_setup,
     };
-    let _g_sip_from_hdr_kw_id = helper_keyword_register_sticky_buffer(&kw);
+    let _g_sip_from_hdr_kw_id = helper_keyword_register_multi_buffer(&kw);
```

Comment:
Why do we bother binding the return value?

Reply:
This PR did not change this...
I guess it can be removed